File: Easy/CompareTheTriplets/src/main/java/MyCode.java
package main.java;

import java.util.*;

public class MyCode {
    public static List<Integer> compareTriplets(List<Integer> a, List<Integer> b) {
        int bobPoints = 0;
        int alicePoints = 0;
        for(int i = 0; i < a.size(); i++){
            if (a.get(i)> b.get(i)){
                alicePoints++;
            }if (a.get(i)<b.get(i)){
                bobPoints++;
            }

        }

        return Arrays.asList(alicePoints,bobPoints);
    }
}

File: Easy/CompareTheTriplets/src/main/kotlin/MyCode.kt
package main.kotlin

fun main(args: Array<String>) {

    val a = readLine()!!.trimEnd().split(" ").map{ it.toInt() }.toTypedArray()

    val b = readLine()!!.trimEnd().split(" ").map{ it.toInt() }.toTypedArray()

    val result = compareTriplets(a, b)

    println(result.joinToString(" "))
}
fun compareTriplets(a:Array<Int>, b :Array<Int>): Array<Int>{
    var alicePoints = 0
    var bobPoints = 0
    for (i in a.indices){
        when{
            a[i] < b[i] -> bobPoints++
            a[i] > b[i] -> alicePoints++
        }
    }
    return arrayOf(alicePoints,bobPoints)

}